### ISSUE TITLE
testing with avdecccmdline, I210 board not visible

### ISSUE THREAD

**Issue by ramagottfried** [audioscience/avdecc-lib#185]
Hello!
Testing the avdecccmdline tool on a ubuntu system, with an I210 board installed -- I am not seeing the I210 port listed in the available interfaces. can someone advise how to fix this?  I tried both the master branch and the staging branch.

The board is already working for Open-AVB talking and listening.

Thanks for any suggestions!

best,
Rama

**Comment by ramagottfried** [comment]
I also tried passing my I210 name as an argument to avdecccmdline with: -i eth0 but get a segfault.  here's the gdb backtrace:

Starting program: /home/rama/avdecc-lib/controller/app/cmdline/avdecccmdline -i eth0
[Thread debugging using libthread_db enabled]
Using host libthread_db library "/lib/x86_64-linux-gnu/libthread_db.so.1".
[New Thread 0x7ffff664a700 (LWP 19897)]
[New Thread 0x7ffff5e49700 (LWP 19898)]
AVDECC Controller version: v0.6.0

Program received signal SIGSEGV, Segmentation fault.
0x00007ffff7416630 in std::string::c_str() const () from /usr/lib/x86_64-linux-gnu/libstdc++.so.6

**Comment by andrew-elder** [comment]
Are you building the staging branch?
I haven't tested linux recently...

**Comment by ramagottfried** [comment]
yes, I tried staging branch and master... any idea why the i210 board wouldn't show up on the list?
does the avdecccmdline program support IPv6?

**Comment by pcub** [comment]
I also have same problem. I have i210 nic card and I am using ubuntu 14.02. 
But wireshark seems capture the frames.

**Comment by RikusW** [comment]
I had the same problem, I assigned an IPv4 address to use telnet and when I tried again sometime later avdecccmdline worked fine. Seems like it needs the IP address to enumerate it.

**Comment by andrew-elder** [comment]
Sounds like this is related to network interface lookup code....
